README.md

```diff
@@ -130,6 +130,13 @@ rhc_proxy:
 Use `{"state":"absent"}` to reset all the proxy configurations to empty
 (effectively disabling the proxy server).
 
+    rhc_environments: []
+
+The list of environments to which register to when connecting the system.
+
+*NB*: this only works when the system is being connected from an unconnected
+state -- it cannot change the environments of already connected systems.
```

Comment:
Can you unregister, then register, if you want to change the environment(s)?

Reply:
Strictly speaking yes, however you will lose everything that was set/configured for that system: repositories configured (both locally and with overrides on the server), the release set, etc.

Not a nice situation, however unfortunately it will require more changes in other places that so far have not been exactly super-important to do :-/